renovate.json

```diff
@@ -17,30 +18,42 @@
       "^config/default/.*-patch\\.yaml$"
     ]
   },
+  "packageRules": [
+    {
+      "matchDepPatterns": [
+        "^.*open-feature/flagd.*$"
+      ],
+      "groupName": "open-feature/flagd"
+    }
+  ],
   "regexManagers": [
     {
-      "fileMatch": ["^apis/core/**/flagsourceconfiguration_types.go$"],
+      "fileMatch": ["^apis/core/.*/flagsourceconfiguration_types.go$"],
       "matchStrings": ["defaultTag\\s*string\\s*= \"(?<currentValue>.*?)\"\\n"],
       "depNameTemplate": "open-feature/flagd",
-      "datasourceTemplate": "github-releases"
+      "datasourceTemplate": "github-releases",
+      "extractVersionTemplate": "^flagd/(?<version>.*)$"
     },
     {
       "fileMatch": ["^chart/open-feature-operator/values.yaml$"],
       "matchStrings": ["repository: \"ghcr\\.io\/open-feature\/flagd\"\\n\\s*tag: (?<currentValue>.*?)\\n"],
       "depNameTemplate": "open-feature/flagd",
-      "datasourceTemplate": "github-releases"
+      "datasourceTemplate": "github-releases",
+      "extractVersionTemplate": "^flagd/(?<version>.*)$"
```

Comment:
I understand every addition except this one, and the doc seems a bit thin. Can you explain this particular setting? Why is it needed?

Reply:
Oh, right. And the tag is prefixed like this now because it's a monorepo. I guess before there was no need for this since release please was only releasing a single artifact?

Reply:
yeah exactly, since that, i think the tags changed